howto/stream/oob-data.md

```diff
@@ -1,14 +1,34 @@
-The exchange of out-of-band data transmission between Android and Anbox (WebRTC platform) enables Android application developers to trigger an action from an Android application and forward it to Anbox(WebRTC platform). Once Anbox receives the action, as one peer of WebRTC platform, the action will be propagated from the Anbox to the remote peer(WebRTC client) through WebRTC data channel so that client can react to the action received from the remote peer and respond accordingly on the UI.
+Enabling out-of-band (OOB) data transmission between an Android application and a WebRTC client makes it possible to exchange data and trigger actions between an Android application and a WebRTC client.
 
-This document will walk you through how to send a message from an Android application
-running in the container to the client application developed with the Anbox Streaming
+Anbox Cloud provides two versions of this OOB data exchange:
+
+- [Version 1](#oob-v1) enables Android application developers to trigger an action from an Android application and forward it to a WebRTC client through the Anbox WebRTC platform. When Anbox receives the action, as one peer of the WebRTC platform, the action is propagated from Anbox to the remote peer (the WebRTC client) through a WebRTC data channel. The client can then react to the action received from the remote peer and respond accordingly on the UI.
```

Comment:
to trigger an action from an Android application
->
 to trigger an action from an Android application running in a container